src/components/security/index.njk

```diff
@@ -16,7 +16,7 @@
       actions: {
         items: [
           {
-            href: enterPasswordUrl + "&type=changeEmail&edit=true",
+            href: enterPasswordUrl + "&type=changeEmail",
```

Comment:
is this correct? Do we not need the edit=true param here (it is the only place in this PR where we are removing it)

Reply:
@saralk, going by this PR where it was removed: https://github.com/govuk-one-login/di-account-management-frontend/pull/1800/files#diff-54b690956706838b48a5df100ac823f3816560712e4728fa673083d47899accf. Essentially what we are doing is re-instating the changes in this PR